cypress/integration/metaNewsSpec.js

```diff
@@ -61,52 +51,58 @@ describe('Article Meta Tests', () => {
     });
   });
 
-  facebookMeta(
-    '100004154058350',
-    '1609039196070050',
-    'https://www.facebook.com/bbcnews',
-  );
-
-  openGraphMeta(
-    'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
-    'https://www.bbc.co.uk/news/special/2015/newsspec_10857/bbc_news_logo.png',
-    'BBC News',
-    'en_GB',
-    'BBC News',
-    "Meghan's bouquet laid on tomb of unknown warrior",
-    'article',
-    `https://www.bbc.com/news/articles/${services.news.pageTypes.articles.asset}`,
-  );
+  it('should have the correct facebook metadata', () => {
+    cy.checkFacebookMetadata(
+      '100004154058350',
+      '1609039196070050',
+      'https://www.facebook.com/bbcnews',
+    );
+  });
 
-  twitterMeta(
-    'summary_large_image',
-    '@BBCNews',
-    'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
-    'BBC News',
-    'https://www.bbc.co.uk/news/special/2015/newsspec_10857/bbc_news_logo.png',
-    '@BBCNews',
-    "Meghan's bouquet laid on tomb of unknown warrior",
-  );
+  it('should have the correct open graph metadata', () => {
+    cy.checkOpenGraphMetadata(
+      'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
+      'https://www.bbc.co.uk/news/special/2015/newsspec_10857/bbc_news_logo.png',
+      'BBC News',
+      'en_GB',
+      'BBC News',
+      "Meghan's bouquet laid on tomb of unknown warrior",
+      'article',
+      `https://www.bbc.com/news/articles/${services.news.pageTypes.articles.asset}`,
+    );
+  });
+  it('should have the correct twitter metadata', () => {
+    cy.checkTwitterMetadata(
+      'summary_large_image',
+      '@BBCNews',
+      'Meghan follows the royal bridal tradition started by the Queen Mother in 1923.',
+      'BBC News',
+      'https://www.bbc.co.uk/news/special/2015/newsspec_10857/bbc_news_logo.png',
+      '@BBCNews',
+      "Meghan's bouquet laid on tomb of unknown warrior",
+    );
+  });
 
   it('should include metadata that matches the JSON data', () => {
-    metadataAssertion();
+    cy.checkArticleMetadata();
```

Comment:
Is this necessary? Isn't this tested in metaArticleSpec?

If it is we can get rid of the command and put all those tests directly in the other file.